test/test_pipeline/test_tabular_classification.py

```diff
@@ -405,6 +384,8 @@ def test_constant_pipeline_iris(fit_dictionary_tabular):
                                              search_space_updates=search_space_updates)
 
     fit_dictionary_tabular['additional_metrics'] = ['balanced_accuracy']
+    # increase number of epochs to test for performance
+    fit_dictionary_tabular['epochs'] = 50
```

Comment:
Repeat from another comment.
Can we remove it or can we use NamedTuple instead? Because it is already defined in the contest and we do not want to change the variable locally. Especially, I am afraid of modifying the value without being recognized by anyone. If it is not appropriate change for this PR, I will raise an issue.